cmd/contour/cli.go

```diff
@@ -45,21 +44,20 @@ func (c *Client) dial() *grpc.ClientConn {
 	case c.CAFile != "" || c.ClientCert != "" || c.ClientKey != "":
 		// If one of the three TLS commands is not empty, they all must be not empty
 		if !(c.CAFile != "" && c.ClientCert != "" && c.ClientKey != "") {
-			log.Fatal("You must supply all three TLS parameters - --cafile, --cert-file, --key-file, or none of them.")
+			kingpin.Fatalf("You must supply all three TLS parameters - --cafile, --cert-file, --key-file, or none of them.")
```

Comment:
This is a pre-existing problem, but could you please remove the capital and fullstops from this message so it formats more consistently?

```suggestion
			kingpin.Fatalf("you must supply all three TLS parameters - --cafile, --cert-file, --key-file, or none of them")
```

Reply:
fixed this.